src/docs/development/tools/sdk/upgrading.md

```diff
@@ -29,36 +26,85 @@ dev_dependencies:
     sdk: flutter
 ```
 
-The `sdk: flutter` line tells the `flutter` command-line tool to find the
-correct package for you.
-
-Do not use the `pub get` or `pub upgrade` commands to manage your dependencies.
+{{site.alert.warning}}
```

Comment:
Good idea to make this a warning (although I'll have to look into why the associated color is red).